### PR TITLE
New expected outputs from tests profileservice.php#13909

### PR DESCRIPTION
**Pull request by b21tilbe**
Updated

### CODE REVIEW

File: backend-models/service-tests/profileservice-tests.md [hunk 1]
```diff
@@ -82,8 +99,17 @@ End pre-
 		$password = "Kong";
```
Comment (a19hakja) [inline backend-models/service-tests/profileservice-tests.md:99]: One thing i noticed here is that you actually have to provide the wrong password to get a response from profileservice.php, as well as the new password has to fulfill the password requirements (min 8 letters, one lowercase, one uppercase and a number) Which is why providing Kong won't do anything with profileservice.php

Comment (a19hakja) [reply backend-models/service-tests/profileservice-tests.md:99]: The output is correct though, i just couldn't get there the info provided here

**Review by a19hakja — CHANGES_REQUESTED** [review]
Otherwise everything seems fine, just change Kong to something that fulfills the requirements and gets a response from profileservice.php. And maybe include a false password that gets us there

**Review by a19hakja — APPROVED** [review]
That did the trick :) I'll approve

**Comment by d21erisu** [comment]
great

**Pull request merged**